<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2160</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>探索模式</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="_table">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>0</y>
      <width>881</width>
      <height>1011</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(234, 251, 255)</string>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>3</height>
     </size>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>180</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
   </widget>
   <widget class="QCalendarWidget" name="_calendar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>1011</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::LongDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_buttonInport">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1030</y>
      <width>160</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>导入课表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_buttonPersonalize">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>1030</y>
      <width>160</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>个性化设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_buttonModify">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>1030</y>
      <width>160</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>重新安排</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_buttonConfirm">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>1030</y>
      <width>160</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>显示地图</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeDisplay">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>1270</y>
      <width>240</width>
      <height>72</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>计时器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_event">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>1130</y>
      <width>160</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>个性化推荐</string>
    </property>
   </widget>
   <widget class="CountDownTimer" name="_countdowntimer" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>1230</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="_save">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>1130</y>
      <width>160</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>同步到本地</string>
    </property>
   </widget>
   <widget class="PKUMap" name="_pkumap" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>20</width>
      <height>1540</height>
     </rect>
    </property>
   </widget>
   <widget class="Introduction" name="_info" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>20</width>
      <height>1540</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CountDownTimer</class>
   <extends>QWidget</extends>
   <header location="global">countdowntimer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PKUMap</class>
   <extends>QWidget</extends>
   <header location="global">pkumap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Introduction</class>
   <extends>QWidget</extends>
   <header location="global">introduction.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
